src/ui/components/DropdownMenu/index.js

```diff
@@ -0,0 +1,33 @@
+/* @flow */
+import React from 'react';
+import classNames from 'classnames';
+
+import Icon from 'ui/components/Icon';
+
+import DropdownMenuItem from './DropdownMenuItem';
```

Comment:
I don't like this style of putting a separate component file inside a component's directory. I understand this is a child component only this component is expected to use, but if that's the case I'd rather see it in the same file.

But I'd rather this be in a separate folder as its own component. It makes it easy to trace a single component to a file/folder and keeps the organisation rules clear/simple.